src/apps/content-editor/src/app/views/ItemEdit/components/Header/ItemVersioning/ItemVersioning.js

```diff
@@ -133,9 +136,13 @@ export function ItemVersioning(props) {
             >
               <Box
                 component="span"
-                sx={{ display: { xs: "none", xl: "block" } }}
+                sx={{ display: { xs: "none", xl: "block" }, marginRight: 0.5 }}
               >
-                Publish Version {props.item.meta.version}
+                Publish
```

Comment:
Couldn't we also just refactor this to:

<Box>Publish {matches ? `Version ${props.item.meta.version}` : ""}</Box>